src/main/scala/cromwell/engine/backend/local/LocalBackendCall.scala

```diff
@@ -35,4 +36,8 @@ case class LocalBackendCall(backend: LocalBackend,
 
   override def useCachedCall(avoidedTo: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] =
     backend.useCachedCall(avoidedTo.asInstanceOf[LocalBackendCall], this)
+
+  override def pollingInitialInterval: FiniteDuration = 10.seconds
+  override def pollingMaxInterval: FiniteDuration = 10.minutes
+  override def pollingMultiplier: Double = 1.1D
```

Comment:
@mcovarr the only thing I can think of without changing the whole `ExecutionActor` logic is giving default "standard" values in the `BackendCall` trait, and override them in the `JesBackendCall` class.
Would that be ok ?